Editor/UI/Stores/PackageStore.cs

```diff
@@ -125,5 +128,20 @@ private void WritePackagesJson()
 			var jsonText = JsonUtility.ToJson(InstalledPackages.Value, true);
 			File.WriteAllText(Settings.AbsolutePackagesJsonFilePath, jsonText);
 		}
+
+		private void HandleCompletPackageListStatusStarted(Action obj)
+		{
+			PackageListStatus.SetValue(null);
+			IsWorking.SetValue(true);
+			Operation.SetValue("Compiling status of each package");
+		}
+
+		private void HandleCompletPackageListStatusComplete(Action obj)
```

Comment:
Wouldn't you prefer to call `obj` `action` instead?

Reply:
Yeah, I didn't choose the name, rider auto generated. I'll changed

Reply:
I have changed it to `action`. Although you should be aware this is not a `System.Action` delegate. It's a flux action, ie: a small object describing changes in state